docs/source/models_new.rst

```diff
@@ -78,6 +149,35 @@ pre-trained weights:
    models/googlenet_quant
    models/mobilenetv2_quant
 
+|
+
+Here is an example of how to use the pre-trained quantized image classification models:
```

Comment:
I wonder if we should just refer to the snippet above instead, and mention to pass `quantize=True`? This seems to be the only difference, apart from the imports.

Reply:
I'm OK giving a second example given we have a different namespace. Hopefully on the future this will be deprecated and move all quantization of models on main model builders (with FX quant)